kufar/books/session.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"os"
+
+	"github.com/gorilla/securecookie"
+	"github.com/gorilla/sessions"
+)
+
+var store *sessions.CookieStore
+
+func init() {
+	authKeyOne := securecookie.GenerateRandomKey(64)
```

Comment:
You don't need to generate it new every time you will launch your application. Just get keys from your envs and its all